hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/Scheduler.java

```diff
@@ -44,18 +45,21 @@
 // TODO (elingg) remove as much logic as possible from Scheduler to clean up code
 public class Scheduler implements org.apache.mesos.Scheduler, Runnable {
 
-  public static final Log log = LogFactory.getLog(Scheduler.class);
-  private final SchedulerConf conf;
+  private final Log log = LogFactory.getLog(Scheduler.class);
+
+  private static final int SECONDS_FROM_MILLIS = 1000;
+
+  private final HdfsFrameworkConfig frameworkConfig;
   private final LiveState liveState;
   private final PersistentState persistentState;
   private final DnsResolver dnsResolver;
 
   @Inject
-  public Scheduler(SchedulerConf conf, LiveState liveState, PersistentState persistentState) {
-    this.conf = conf;
+  public Scheduler(HdfsFrameworkConfig frameworkConfig, LiveState liveState, PersistentState persistentState) {
```

Comment:
We should be consistent with our naming and name this hdfsFrameworkConfig as its named elsewhere.